angularcapture.py

```diff
@@ -0,0 +1,91 @@
+import nidaqmx
+from nidaqmx.stream_readers import CounterReader
+from nidaqmx.constants import EncoderType, EncoderZIndexPhase, AngleUnits
+import numpy as np
+import click
+
+
+@click.command()
+@click.argument('samples', type=click.INT)
+@click.argument('encodertype')
```

Comment:
https://click.palletsprojects.com/en/7.x/options/
Could you change to use click.Choice?